src/firefly/js/tables/ui/TableRenderer.js

```diff
@@ -337,7 +346,8 @@ export function ContentEllipsis({children, text, style={}, actions=[]}) {
 
     const onActionsClicked = (ev) => {
         ev.stopPropagation();
-        showDropDown({id: dropDownID, content: dropDown, atElRef: actionsEl.current, locDir: 33, style: {marginLeft: -10},
+        const boxEl = findTableFor(actionsEl.current);
```

Comment:
My first instinct to make ellipsis dropdown go away was to click on the table toolbar but this only lets clicks inside table (rows and headers) to hide dropdown. Generally, dropdowns go away by clicking anywhere on the page and I can see you using `document` as fallback for `boxEl` prop. But based on your slack message, I'm guessing document level clicks won't work because other firefly components are stopping event propagation?

That being said, this PR still fixes the issue and we can control dropdown behavior more carefully in near future when migrating to joy UI.